Tasks/ExtractFiles/extractfilestask.ts

```diff
@@ -7,7 +7,7 @@ var archiveFilePatterns: string[] = tl.getDelimitedInput('archiveFilePatterns',
 var destinationFolder: string = path.normalize(tl.getPathInput('destinationFolder', true, false).trim());
 var cleanDestinationFolder: boolean = tl.getBoolInput('cleanDestinationFolder', false);
 
-var repoRoot: string = tl.getVariable('build.sourcesDirectory');
+var repoRoot: string = tl.getVariable('System.DefaultWorkingDirectory');
```

Comment:
I confirmed in testing and with Eric Sciple that System.DefaultWorkingDirectory is available in both Build and Release.